docker/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+cp docker/docker_settings.py knesset/local_settings.py && \
+python manage.py migrate && \
+python manage.py createsuperuser --noinput --username=superuser --email=superuser@example.com && \
```

Comment:
* do the migrate and createsuperuser only if DB doesn't exist (was downloaded fresh)